frontend/lib/ui/ProgressBar/ProgressBar.tsx

```diff
@@ -2,28 +2,88 @@ import { cn } from "@kiesraad/util";
 
 import cls from "./ProgressBar.module.css";
 
+export type ProgressBarColorClass =
+  | "default"
+  | "errors_and_warnings"
+  | "unfinished"
+  | "in_progress"
+  | "first_entry_finished"
+  | "definitive"
+  | "not_started";
+
+export type RadiusClass = "no-radius" | "default-radius" | "radius-right" | "radius-left";
+export type PercentageAndColorClass = {
+  percentage: number;
+  class: ProgressBarColorClass;
+};
+
 export interface ProgressBarProps {
-  percent: number;
   id: string;
+  percentagesAndColorClasses: PercentageAndColorClass[];
```

Comment:
From a datavis side, I would prefer to name this "data" and perhaps just do a tuple of [pct, color]
```
data: [number, ProgressBarColorClass] | [number, ProgressBarColorClass][]
```

Reply:
Changed in c3454c192dbc8c694a1c85fce5b8ed5ee45d7605